app/validators/blocks/block_validator.py

```diff
@@ -10,7 +13,7 @@ class BlockValidator(Validator):
         "Placeholder references an answer in the same block (self-reference)"
     )
 
-    def __init__(self, block_element, questionnaire_schema):
+    def __init__(self, block_element, questionnaire_schema: QuestionnaireSchema):
```

Comment:
I know it's a detail but can we type hint both arguments here (since we did one)?

Reply:
Sure 👍